src/reducers/sources.js

```diff
@@ -69,14 +72,14 @@ function update(state = State(), action: Action) : Record<SourcesState> {
       });
 
     case "CLOSE_TAB":
-      availableTabs = removeSourceFromTabList(state.tabs, action.id);
+      availableTabs = removeSourceFromTabList(state, state.tabs, action.id);
```

Comment:
Hmm, I'm worried about selecting source because that also loads them and we actually only want to load the active tab